bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Eclipse/Generator.cpp

```diff
@@ -0,0 +1,117 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Astrodynamics/Eclipse/Generator.hpp>
+
+inline void OpenSpaceToolkitAstrodynamicsPy_Eclipse_Generator(pybind11::module& aModule)
+{
+    using namespace pybind11;
+
+    using ostk::core::container::Array;
+    using ostk::core::type::Shared;
+    using ostk::core::type::String;
+
+    using ostk::physics::Environment;
+    using ostk::physics::environment::utilities::Eclipse;
+    using ostk::physics::time::Duration;
+    using ostk::physics::time::Interval;
+
+    using ostk::astrodynamics::eclipse::Generator;
+    using ostk::astrodynamics::Trajectory;
+
+    class_<Generator, Shared<Generator>>(
```

Comment:
```suggestion
    class_<Generator>(
```
Don't think you need to have the Shared<Generator> here as it's a base class and not inherited, not used as a shared pointer across our repo